src/main/resources/swagger.api/schemas/publication_schema.yaml

```diff
@@ -72,3 +72,29 @@ PublicationStatusError:
     errorCode:
       description: error code of failed request
       type: integer
+
+PublicationResultCollection:
+  description: "A JSON schema for the publication result collection"
+  type: object
+  properties:
+    publicationResults:
+      type: array
+      description: "The list of publication results"
+      items:
+        type: object
+        $ref: "publication_schema.yaml#/PublicationResult"
+    totalRecords:
+      type: integer
+
+PublicationResult:
+  description: publication result
```

Comment:
there's usage of quotes in part of places and in others there's not, we should pick some common approach

Reply:
https://issues.folio.org/browse/MODCON-76
created a separate ticket where we can decide how to deal with such issues